pkgs/by-name/ig/igir/package.nix

```
@@ -2,46 +2,47 @@
   lib,
   buildNpmPackage,
   fetchFromGitHub,
-
-  # for patching bundled 7z binary from the 7zip-bin node module
-  # at lib/node_modules/igir/node_modules/7zip-bin/linux/x64/7za
-  autoPatchelfHook,
   stdenv,
+  SDL2,
+  autoPatchelfHook,
 }:
-
 buildNpmPackage rec {
   pname = "igir";
-  version = "2.11.0";
+  version = "3.0.1";
 
   src = fetchFromGitHub {
     owner = "emmercm";
     repo = "igir";
     rev = "v${version}";
-    hash = "sha256-NG0ZP8LOm7fZVecErTuLOfbp1yvXwHnwPkWTBzUJXWE=";
+    hash = "sha256-QPUnvirufY3UiSFVxX3xCmpAlzPKg5JxwAnVefdepqU=";
   };
 
-  npmDepsHash = "sha256-ADIEzr6PkGaJz27GKSVyTsrbz5zbud7BUb+OXPtP1Vo=";
+  npmDepsHash = "sha256-hJzgFWUJnEQu+EGD4itv04IQl7Rwm8CYeFELJ9aoTMY=";
 
   # I have no clue why I have to do this
   postPatch = ''
     patchShebangs scripts/update-readme-help.sh
   '';
 
-  nativeBuildInputs = [ autoPatchelfHook ];
+  buildInputs = [
+    (lib.getLib stdenv.cc.cc)
+    SDL2
+  ];
 
-  buildInputs = [ (lib.getLib stdenv.cc.cc) ];
+  nativeBuildInputs = lib.optional stdenv.hostPlatform.isLinux [ autoPatchelfHook ];
```

You should either use `lib.optionals` here or drop the brackets:

```suggestion
  nativeBuildInputs = lib.optionals stdenv.hostPlatform.isLinux [ autoPatchelfHook ];
```

See:
- [lib.optional - Nix function reference](https://noogle.dev/f/lib/optional)
- [lib.optionals - Nix function reference](https://noogle.dev/f/lib/optionals)